### DISCUSSION TITLE
Amazon Alexa

### DISCUSSION THREAD

**Discussion by hide-mukai** [amberelectric/public-api#85] [General]
Hello,

I want to understand if it is possible to get my Amazon Alexa to interact and obtain price data, etc.
Simple use case is that I can ask Amazon Alexa for the current price (Just so I don't have to open the Amber app on phone and check manually)
There may be basic knowledge that I don't have in order to achieve this.
I would appreciate anyone providing some general guidance/external links for researching.

Thanks,
Hide

**Comment by madpilot** [comment]
It's on our roadmap! I don't have an ETA at the moment though. Though it'll be the next third party integration we'll do (probably)

I'll add this as another cite though - every time someone asked it gets pushed up the priority list a little.

**Reply by madpilot** [reply]
If you have got some programmer chops, you can also build it yourself
